geps/gep-1742/index.md

```diff
@@ -357,9 +357,9 @@ sequenceDiagram
     P->>U: Finishes Headers
```

Comment:
I just noticed that I think we need to swap these two lines, the headers should finish before the request overall.

Reply:
You are right, I swapped them.